scripts/gas-refund-program/transactions-indexing/successful-swap-tx-indexing.ts

```diff
@@ -101,27 +99,21 @@ export async function computeSuccessfulSwapsTxFeesRefund({
     );
 
     // alternatively can slice requests over different sub intervals matching different stakers subset but we'd be refetching same data
-    const swaps = await getSuccessfulSwaps({
+    const txs = await getAllTXs({
+      epoch,
       startTimestamp: _startTimestampSlice,
       endTimestamp: _endTimestampSlice,
       chainId,
     });
 
     logger.info(
-      `fetched ${swaps.length} swaps between ${_startTimestampSlice} and ${_endTimestampSlice}`,
+      `fetched ${txs.length} txs between ${_startTimestampSlice} and ${_endTimestampSlice}`,
     );
 
     const updatedPendingGasRefundDataByAddress: TxFeesByAddress = {};
 
     await Promise.all(
-      swaps.map(async swap => {
-        if (
-          epoch >= GasRefundTxOriginCheckStartEpoch &&
-          swap.initiator !== swap.txOrigin
-        ) {
-          return;
-        }
-
+      txs.map(async swap => {
```

Comment:
Careful txOrigin check has been deleted here

Reply:
yeah I just moved it [here](https://github.com/paraswap/paraswap-volume-tracker/pull/35/files#diff-a5e803cc7c0928f1758c80d477480d55eb47eab03ac52e74b171d1ad803f672cR79) instead.

Reply:
This doesn't need to be async anymore